src/package.json

```diff
@@ -26,6 +26,7 @@
     "babel-register": "6.24.1",
     "babel-require": "1.0.1",
     "electron": "^1.7.9",
+    "md5": "^2.2.1",
```

Comment:
We should be using `save-exact=true`. We should probably add that later, but for now I think just removing the carets should be good enough.

Reply:
Looks like we do have save-exact=true in the .npmrc. Not sure how these got in there. Good catch.